packages/core/src/utils/find-parent-node-of-type.ts

```diff
@@ -12,13 +12,12 @@ export function findParentNodeOfType(
   /**
    * The type of the node to remove.
    */
-  type: NodeType | string,
+  types: NodeType | string | Array<NodeType | string>,
```

Comment:
1. Let's not introduce API breaking changes unless we have to.
2. Is this change related to the `table-handle` feature? I did a quick search and couldn't find its usage in this PR.  If this change is not required by the `table-handle` feature, it's better to open it in another PR.

Reply:
I'm sorry, but this is the function I used before, and it was deleted in the final state. I'm ready to restore it to its original state.